plugins/extractors/mysql/mysql.go

```diff
@@ -31,31 +31,10 @@ var defaultDBList = []string{
 	"sys",
 }
 
-// Config holds the connection URL for the extractor
-type Config struct {
-	ConnectionURL string `mapstructure:"connection_url" validate:"required"`
-}
-
 var sampleConfig = `connection_url: "admin:pass123@tcp(localhost:3306)/"`
 
-// Extractor manages the extraction of data from MySQL
-type Extractor struct {
-	excludedDbs map[string]bool
-	logger      log.Logger
-	config      Config
-	db          *sql.DB
-	emit        plugins.Emit
-}
-
-// New returns a pointer to an initialized Extractor Object
-func New(logger log.Logger) *Extractor {
-	return &Extractor{
-		logger: logger,
-	}
-}
-
 // Info returns the brief information about the extractor
-func (e *Extractor) Info() plugins.Info {
```

Comment:
And after these I am confused on how to use receiver (e *Extractor) cause it was throwing error when I tried using (e *extutil.Extractor) since Extractor type is no longer declared in the same package. So I am not sure on whether I should remove it or if there is some other way around.